### ISSUE TITLE
The latest release v3.0.73 not work on alpine

### ISSUE THREAD

**Issue by k8scat** [aliyun/aliyun-cli#344]
Install aliyun-cli in `python:3.7-alpine`:

```sh
apk add --no-cache curl
curl -LO https://github.com/aliyun/aliyun-cli/releases/download/v3.0.73/aliyun-cli-linux-3.0.73-amd64.tgz
tar zxf aliyun-cli-linux-3.0.73-amd64.tgz
```

And it does not work:

```sh
./aliyun
sh: ./aliyun: not found
```

I have tried to build from source code (commit: 2fb764bbd4f916471f10fcfc66b64bbe2fdd27aa) and it work correctly in alpine.

**Comment by k8scat** [comment]
I noticed that a new version has been released (v3.0.74), but this problem is still not solved:

```sh
# ./aliyun
sh: ./aliyun: not found
```

**Comment by hayorov** [comment]
Hello @k8scat ,

The output "X not found" is about missed dynamically linked library. (Probably the most confusing and ambiguous error message).

run `ldd aliyun` to get more diagnostics.

## The solution
For alpine add this into your Dockerfile
```
RUN apk add --no-cache libc6-compat musl
```
